dashboard/src/components/dialogs/binding/node-binding-dialog.ts

```diff
@@ -298,17 +335,42 @@ export class NodeBindingDialog extends LitElement {
               <div class="group-label">target</div>
               <md-outlined-text-field
                 label="node id"
+                name="NodeId"
+                type="number"
+                min="0"
+                max="65535"
                 class="target-item"
+                @change=${this.onChange}
+                supporting-text="required"
               ></md-outlined-text-field>
               <md-outlined-text-field
                 label="endpoint"
+                name="Endpoint"
+                type="number"
+                min="0"
+                max="65535"
+                @change=${this.onChange}
                 class="target-item"
+                supporting-text="required"
               ></md-outlined-text-field>
               <md-outlined-text-field
                 label="cluster"
+                name="Cluster"
+                type="number"
+                min="0"
+                max="65535"
+                @change=${this.onChange}
                 class="target-item"
+                supporting-text="optional"
               ></md-outlined-text-field>
             </div>
+            <div style="margin:8px;">
+              <p style="font-size: 10px;font-style: italic;font-weight: bold;">
+                Note: When the Cluster is empty, the binding will be applied to
```

Comment:
Please use the text proposed by @agners  Thanks

Reply:
I would still favor this because of the content. We will simplify that later again when matter.js is in